index.js

```diff
@@ -188,7 +164,7 @@ function sendNotification(endpoint, params) {
         options.headers['Authorization'] = 'Bearer ' + jwt;
         var key = 'p256ecdsa=' + urlBase64.encode(vapid.publicKey);
         if (options.headers['Crypto-Key']) {
-          options.headers['Crypto-Key'] += ',' + key;
+          options.headers['Crypto-Key'] += ';' + key;
```

Comment:
My only counter argument is:

> one with the a "p256dh" key and another with a "p256ecdsa" key

They are are two (key, value) pairs which I'd view as two parameters. This is confused by:

>  this results in two values

Seems like a spec ambiguity @beverloo + @martinthomson